.travis.yml

```diff
@@ -0,0 +1,2 @@
+install: true
+language: scala
```

Comment:
For me, it looks fine.

Sent from a mobile device. Sorry for brevity.
On 25 Feb 2015 09:39, "Merlin Nimier-David" <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/merlinND/TradingSimulation/pull/1#discussion_r25324748>
> :
>
> > @@ -0,0 +1,2 @@
> > +install: true
> > +language: scala
>
> I don't like this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/merlinND/TradingSimulation/pull/1/files#r25324748>.
>